app/Post_Types/Diviner_Field/AdminModifications.php

```diff
@@ -92,37 +131,144 @@ function rc_scd_create_dashboard() {
 		require_once( ABSPATH . 'wp-admin/admin-header.php' );
 		?>
 
-		<div class="wrap wrap-diviner wrap-diviner--default">
-			<?php
-			$defaultTable = new Default_Fields_List_Table();
-			$defaultTable ->prepare_items();
-			?>
-			<h2>Default Fields</h2>
-			<div class="about-text">
-				<?php _e('Donec id elit non mi porta gravida at eget metus. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.' ); ?>
+		<div class="wrap wrap-diviner wrap-diviner--limited wrap-diviner--default">
+
+			<h2>Select a Field Type</h2>
+
+			<p>
+				So you have a bunch of photos/videos/documents….but how do you want your audience to be able to find them? Whether it’s by date, location, or subject matter, this is where you simultaneously
+			</p>
+
+			<ul>
+				<li>
+					Design your uploading experience by choosing what information will be assigned to each archive item, and
+				</li>
+				<li>
+					Design the browse page of your archive.
+				</li>
+			</ul>
+
+			<p>
+				There are five kinds of fields:
+			</p>
+
+
+			<div class="field-select-wrap">
+				<h2>Text Field</h2>
+				<p>
+					<?php _e('Add a text field for a type of information you wish to assign to EACH archive item, but which will be completely different for each archive item. Example: serial number, catalog number, internal title, etc.' ); ?>
+				</p>
+				<a href="post-new.php?post_type=diviner_field&field_type=<?php echo CPT_Field::NAME; ?>" class="button button-primary button-hero">
```

Comment:
ideally, in all these slugs `post_type=diviner_field` should be replaced by the use of a constant as field name is; this is not a big issue though.

Reply:
Done